src/main/java/core/basesyntax/FileWrite.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.io.File;
+
+public interface FileWrite {
```

Comment:
```suggestion
public interface FileWriter {
```
no verbs should be in interface name